truncate/table_schema_test.go

```diff
@@ -0,0 +1,225 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package truncate
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
+func TestFilterTableSchemas(t *testing.T) {
```

Comment:
In Go, it's common to have different test func for different package function, so maybe it'd better to have `TestFilterTableSchemas`, `TestTargetFilterTableSchemas`, and `TestExcludeFilterTableSchemas`.

The `filterTableSchemas` function is very straightforward, so I think it's ok to skip wrighting`TestFilterTableSchemas`.